zebrad/src/components/sync/downloads.rs

```diff
@@ -169,6 +182,16 @@ where
                     rsp = block_req => rsp?,
                 };
 
+                // Check if the block is already in the state.
+                // BUG: check if the hash is in any chain (#862).
+                // Depth only checks the main chain.
```

Comment:
"main chain" can be confusing, because it sounds like "Mainnet"
```suggestion
                // Depth only checks the best chain.
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/2327/commits/b3660f86565ded51382e109cf7aa9f156f9b0516